src/EventBand/Transport/AmqpLib/AmqpLibDriver.php

```diff
@@ -62,7 +68,7 @@ protected function getConnection()
     /**
      * Get channel
      *
-     * @return AMQPChannel
+     * @return \PhpAmqpLib\Channel\AMQPChannel
```

Comment:
Do you really need a full path here? IDE should work correctly, don't know about phpdoc, i guess should too